src/components/pipeline/PipelineAccordion.js

```diff
@@ -0,0 +1,100 @@
+import {
+  Accordion,
+  AccordionActions,
+  AccordionDetails,
+  AccordionSummary,
+  Button,
+  Container,
+  Divider,
+  ListItemIcon,
+  Typography,
+} from '@material-ui/core';
+import AddIcon from '@material-ui/icons/Add';
+import { makeStyles } from '@material-ui/core/styles';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import CodeIcon from '@material-ui/icons/Code';
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import PropTypes from 'prop-types';
+
+const useStyles = makeStyles((theme) => ({
+  wrapper: {
+    width: '100%',
+  },
+  formControl: {
+    margin: theme.spacing(1),
+    minWidth: 150,
+  },
+  addAccordion: {
+    fontSize: theme.typography.pxToRem(15),
+    fontWeight: theme.typography.fontWeightRegular,
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+}));
+
+const PipelineAccordion = ({ pipelineAlgorithms }) => {
+  const { t } = useTranslation();
+  const classes = useStyles();
+  const [expanded, setExpanded] = useState(false);
+
+  const handleAccordionToggle = (panel) => (event, isExpanded) => {
+    setExpanded(isExpanded ? panel : false);
+  };
+
+  return (
+    <Container>
+      <Typography variant="h6">{t('Pipeline Schema')}</Typography>
+      <div className={classes.wrapper}>
+        {pipelineAlgorithms?.map((algorithm, i) => {
+          return (
+            <Accordion
+              expanded={expanded === `panel${i + 1}`}
+              onChange={handleAccordionToggle(`panel${i + 1}`)}
+            >
+              <AccordionSummary
+                expandIcon={<ExpandMoreIcon />}
+                aria-controls="panel1a-content"
+                id="panel1a-header"
```

Comment:
Won't this make every `algorithm` you are mapping from `pipelineAlgorithms` have the same `aria-controls` and `id`?